src/openms/source/FORMAT/MzMLFile.cpp

```diff
@@ -148,6 +148,11 @@ namespace OpenMS
     Internal::MzMLHandler handler(map, filename, getVersion(), *this);
     handler.setOptions(options_);
     safeParse_(filename, &handler);
+
+    if (options_.getSkipChromatograms())
+    {
+      map.getChromatograms().clear();
```

Comment:
I think this is a suboptimal solution as it will still parse the chromatograms.
skipping should be done in the parser